messaging/kafkaSSL/pom.xml

```diff
@@ -31,11 +31,6 @@
             <artifactId>kafka</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
```

Comment:
There is one more occurence at https://github.com/quarkus-qe/quarkus-test-suite/blob/main/messaging/infinispan-grpc-kafka/pom.xml#L46.